src/ape/pytest/fixtures.py

```diff
@@ -1,14 +1,18 @@
-from typing import List
+from typing import Iterator, List
 
 import pytest
 
 from ape.api import TestAccountAPI
+from ape.logging import logger
 from ape.managers.chain import ChainManager
 from ape.managers.project import ProjectManager
 from ape.utils import ManagerAccessMixin
 
 
 class PytestApeFixtures(ManagerAccessMixin):
+    def __init__(self):
+        self._warned_for_unimplemented_snapshot = False
```

Comment:
At first I thought this said "warn for unimplemented snapshot", would flipping the parity of this make it easier to read?

Reply:
i've no clue lol, whatever seems best I'll rename it as (i've got no attachment)